crates/ruma-events/src/room/message/url_preview.rs

```diff
@@ -0,0 +1,397 @@
+use serde::{Deserialize, Serialize};
+
+use crate::room::{EncryptedFile, OwnedMxcUri, UInt};
+
+/// The Source of the PreviewImage.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub enum PreviewImageSource {
+    #[serde(rename = "beeper:image:encryption")]
```

Comment:
Is there a reason this does not have the alias?

Reply:
yes. for reasons I do not understand, adding `alias = "matrix:image:encryption"` will not make it match on
```
            "m.url_previews": [
            {
                "matrix:matched_url": "https://matrix.org",
                "og:title": "Matrix.org",
                "og:url": "https://matrix.org/",
                "og:description": "Matrix, the open protocol for secure decentralised communications",
                "matrix:image:size": 16588,
                "og:image:height": 400,
                "og:image:type": "image/jpeg",
                "og:image:width": 800,
                "matrix:image:encryption": {
```

I've added the test as an `ignore`, if you want to check...

Reply:
Ah right, is seems that when using `flatten` on an enum, the `alias` is ignored: https://github.com/serde-rs/serde/issues/2188.

Let's remove it and the test for now then.